packages/server/src/time-control/time-control.ts

```diff
@@ -59,6 +62,31 @@ export function ValidateTimeControlBase(
   );
 }
 
+/**
+ * Returns a new time handler instance suited for a given variant.
+ */
+export function GetTimeHandler(
```

Comment:
nit - we should probably figure out what we want our naming conventions to be, but usually functions are lowerCamel (whereas UpperCamel is reserved for classes and types)

Reply:
Yeah I've been inconsistent, sorry about that. At work we have different technology stacks for frontend and backend, and the style guidelines also differ. So I'm used to e.g. write upper case in the backend.

Anyway, I renamed the function.

Reply:
How do we want to write variables, by the way? I've seen Snake-Case, but I've also used Camel-Case.

Reply:
I've been super inconsistent about this unfortunately. From a brief web search, it seems like lowerCamel might be more common for JS, generally.

- https://github.com/airbnb/javascript?tab=readme-ov-file#naming--camelCase
- https://google.github.io/styleguide/jsguide.html#naming-local-variable-names
- https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Writing_style_guide/Code_style_guide/JavaScript#variables